src/core/GCAudio.qml

```diff
@@ -100,14 +100,6 @@ Item {
      */
     signal done
 
-    property var muteChangeHandler: function () { muted ? volume = 0 : volume = 1 }
-
-    /**
-     *  When mute is changed we set the volume to 0 to mute a potential playing
-     * sound.
-     */
-    onMutedChanged: muteChangeHandler()
```

Comment:
I don't think there's any more need for these two statements, as now onMutedChanged signal can directly be used externally wherever required.